lib/gooddata/models/project.rb

```diff
@@ -624,7 +624,8 @@ def attributes_by_title(title)
     # @return [GoodData::ProjectRole] Project role if found
     def blueprint(options = {})
       options = { include_ca: true }.merge(options)
-      result = client.get("/gdc/projects/#{pid}/model/view", params: { includeDeprecated: true, includeGrain: true, includeCA: options[:include_ca] })
+      includeDeprecated = options[:include_deprecated] || true
+      result = client.get("/gdc/projects/#{pid}/model/view", params: { includeDeprecated: includeDeprecated, includeGrain: true, includeCA: options[:include_ca] })
```

Comment:
- model/view must always set includeDeprecated=true as previous.
- we only set includeDeprecated=true if param[includeDeprecated]=true in model/diff when release and rollout brick (do not change other usecases). Please update unit tests to cover 2 bricks release and rollout with/without param includeDeprecated